lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -60,6 +60,7 @@ sub init {
     $self->{AddVolumeGroupPage}            = Installation::Partitioner::LibstorageNG::v4_3::AddVolumeGroupPage->new({app => YuiRestClient::get_app()});
     $self->{AddLogicalVolumePage}          = Installation::Partitioner::LibstorageNG::v4_3::AddLogicalVolumePage->new({app => YuiRestClient::get_app()});
     $self->{ResizePage}                    = Installation::Partitioner::LibstorageNG::v4_3::ResizePage->new({app => YuiRestClient::get_app()});
+    $self->{RolePage}                      = Installation::Partitioner::LibstorageNG::v4_3::RolePage->new({app => YuiRestClient::get_app()});
```

Comment:
I also wanted to place all the variables and methods alphabetically in the file, but this will artificially increase completeness of the PR and also I know that some work is doing in parallel on this file, so I'll do that in the separate PR.